contracts/FractalUsul.sol

```diff
@@ -0,0 +1,56 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+// Usul was previously named "Seele" and SekerDAO was TokenWalk
+// that's where this naming differences are coming from
+import "@tokenwalk/seele/contracts/Usul.sol";
+
+contract FractalUsul is Usul {
+  struct Transaction {
+    address to;
+    uint256 value;
+    bytes data;
```

Comment:
@mudrila what does data this look like when it is emitted?

Reply:
Ah yes, so we should be able to use the Gnosis Safe API to decode it

![image](https://user-images.githubusercontent.com/84364476/204644210-aee9010d-ccd0-4f3a-abce-f7bc1b48d642.png)